db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/CompilerContext.java

```diff
@@ -183,7 +183,7 @@
     NativeSparkModeType DEFAULT_SPLICE_NATIVE_SPARK_AGGREGATION_MODE = NativeSparkModeType.SYSTEM;
     boolean DEFAULT_SPLICE_ALLOW_OVERFLOW_SENSITIVE_NATIVE_SPARK_EXPRESSIONS = true;
     int DEFAULT_SPLICE_CURRENT_TIMESTAMP_PRECISION = 6;
-    int DEFAULT_TIMESTAMP_PRECISION = 9;
+    String DEFAULT_TIMESTAMP_FORMAT = "yyyy-MM-dd HH:mm:ss.SSSSSSSSS";
```

Comment:
The current maximum on-disk timestamp precision is 6 decimal places.  Maybe we should make the default timestamp format match this.  Having a default format with 9 decimal places might make users mistakenly think they can input such data into tables as well.

Reply:
agreed